odkx-src/xlsx-converter-reference.rst

```diff
@@ -374,6 +375,9 @@ The following prompt types are available in ODK-X Survey.
       | to the user, and allows the user to click one item. One of the
       | choices provided is an other option which if clicked provides a text
       | box for the user to enter a value.
+  * - signature
+    - | Used to capture the user signature and allows them to enter a signature
```

Comment:
> user signature

It is not necessarily the user's signature, it could be anyone's signature. I think the docs shouldn't specify whose signature it captures to avoid confusion.

Reply:
Agree with @linl33. I think "Used to capture a signature" or "Used to enter a signature" is simpler/clearer.

Reply:
Okay, will make the changes.